src/mscorlib/src/System/Lazy.cs

```diff
@@ -18,6 +18,7 @@
 using System.Runtime;
 using System.Runtime.InteropServices;
 using System.Security;
+using System.Security.Permissions;
```

Comment:
@jkotas 

Yeah, sorry about that I'm bashing around trying to control git (I'm used to mercurial at work, and I must say, beyond rudimentary commands git drives me insane!)

Anyway, I'll get there...